src/display/annotation_helper.js

```diff
@@ -98,14 +98,14 @@ var AnnotationUtils = (function AnnotationUtilsClosure() {
       }
 
       // Border color
-      if (item.color) {
+      if (item.color !== null) {
```

Comment:
Do you actually need the `!== null` check? Wouldn't the existing `if (item.color) {` suffice, since even an empty array should pass be that check?
*The same question applies below as well.*